_data/sidebar_doc.yml

```diff
@@ -495,8 +495,11 @@ entries:
     - title: Releases
       items:
 
-        - title: v1.0-rc.2
-          url: /v1.0-rc.2.html
+        - title: 1.0
+          url: /v1.0.html
+
+        - title: Known Limitations
```

Comment:
We're sticking the release notes and limitations into the "releases" sidebar? That seems a bit odd, no?